code/controllers/subsystem/air.dm

```diff
@@ -44,6 +45,12 @@ SUBSYSTEM_DEF(air)
 	var/map_loading = TRUE
 	var/list/queued_for_activation
 
+	// VORELINDA config options (am I really calling it that?)
```

Comment:
> VORELINDA

Any other name that doesn't explicitly reference that weird fetish, like LINDA+, V-LINDA, PUT-LINDA, P-LINDA, CIT-LINDA, would be appreciated.

Reply:
LINDA-next